include/sys/Makefile.am

```diff
@@ -9,6 +9,7 @@ COMMON_H = \
 	avl.h \
 	avl_impl.h \
 	bitops.h \
+	blake3.h \
```

Comment:
I think you probably also need to add zfs_chksum.h in this file for it to build nicely.

Reply:
Added, thanks for the hint.
Could you test the arm64-neon speed ?

Reply:
Well, my test Pi4 panicked when I loaded the module, but that might have been for having mixed module versions or god knows what else - it was running CentOS 7 for testing something else. I'll be home in a day or two and able to reboot and test it again then.

Reply:
> Well, my test Pi4 panicked when I loaded the module, but that might have been for having mixed module versions or god knows what else - it was running CentOS 7 for testing something else. I'll be home in a day or two and able to reboot and test it again then.

Oh, I am sorry for this :-/
I used fletcher-4 in the benchmarks also... it seems that fletcher-4 isn't initilized properly in that state. I removed it and the tests are running fine again now.

Reply:
I would suspect you need to be sure fletcher_4_init() is called first, which normally happens in zcommon_init().